100ms is not enough for some responses

I am using Watson Conversation in my bot and 100ms timeout (hardcoded [here](https://github.com/gratifyguy/botkit-mock/blob/ee7817318c1017bcded3b2c9b17d104e9ce577d1/lib/BotmockWorker.js#L96)) is not enough.
Manually editing the file to change timeout to 1000ms makes it work, but it makes all tests slower so it isn't a good solution.
Also I want to do some processing which takes multiple seconds before sending the response.

Would it be possible to handle messages with callback on reply instead of setting a timeout? That way quick responses would be processed quickly and slow responses would get more time without making everything slow.

Setting timeout property in the message does not help.
The test take longer, but fails in the same way:
`AssertionError: undefined == 'I heard a test'`

--- a/lib/BotmockWorker.js
+++ b/lib/BotmockWorker.js
@@ -93,11 +93,11 @@ function BotmockWorker(botkit, config){
                             }else{
                                 cb(null)
                             }
-                        }, 100);
+                        }, message.timeout || 100);
                     };
 
                     //add default timeout for async workflow
-                    setTimeout(logic, message.timeout || 100)
+                    setTimeout(logic, 0)
                 })
             });
         });

